packages/insomnia-app/app/ui/components/modals/workspace-environments-edit-modal.js

```diff
@@ -329,35 +329,22 @@ class WorkspaceEnvironmentsEditModal extends React.PureComponent<Props, State> {
   }
 
   async _handleClickColorChange(environment: Environment) {
-    let el = document.querySelector('#env-color-picker');
-
-    // Remove existing child so we reset the event handlers. This
-    // was easier than trying to clean them up later.
-    if (el && el.parentNode) {
-      el.parentNode.removeChild(el);
-    }
-
-    el = document.createElement('input');
-    el.id = 'env-color-picker';
-    el.type = 'color';
-    document.body && document.body.appendChild(el);
-
     const color = environment.color || '#7d69cb';
-
     if (!environment.color) {
       await this._handleChangeEnvironmentColor(environment, color);
     }
 
-    el.setAttribute('value', color);
-    el.addEventListener('input', (e: Event) => {
-      if (e.target instanceof HTMLInputElement) {
-        this._handleChangeEnvironmentColor(environment, e.target && e.target.value);
-      }
-    });
-
+    const el = document.querySelector('#env-color-picker');
+    el.value = color;
     el.click();
   }
 
+  _handleInputColorChange(environment: Environment, event: Event) {
+    if (event.target instanceof HTMLInputElement) {
+      this._handleChangeEnvironmentColor(environment, event.target && event.target.value);
+    }
+  }
```

Comment:
This event handler can be simplified...

```suggestion
  _handleInputColorChange(e: SyntheticEvent<HTMLInputElement>) {
      this._handleChangeEnvironmentColor(this._getActiveEnvironment(), event.target && event.target.value);
  }
```

Then on the input itself, consumed as:
```js
<input
  ...
  type="color"
  onInput={this._handleInputColorChange}
/>
```

Reply:
> input is fired on the <input> element every time the color changes. The change event is fired when the user dismisses the color picker. In both cases, you can determine the new value of the element by looking at its value.
[source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color)

I've tried to use `onChange` and we keep the same behavior 🤔 
![2647-onChange-event](https://user-images.githubusercontent.com/4171593/96831463-6da4a780-143d-11eb-8a9f-d239c76eb26e.gif)

Therefore we can change the event 🤷